pkg/api/graph/graph.go

```diff
@@ -505,10 +506,10 @@ func ReverseGraphEdge(g Interface, head, tail graph.Node, edgeKinds sets.String)
 	return false
 }
 
-// AddReversedEdge adds a reversed edge for every passed edge and preserves the existing
-// edge. Used to convert a one directional edge into a bidirectional edge, but will
-// create duplicate edges if a bidirectional edge between two nodes already exists.
-func AddReversedEdge(g Interface, head, tail graph.Node, edgeKinds sets.String) bool {
+// AddReversedEdge adds a reference edge from head to tail without changing exiting edges.
```

Comment:
Corollary: in my opinion this method `AddReversedEdge` (of which I am the only current consumer) should just have this signature:
```go
func AddReversedEdge(g Interface, edge graph.Edge) bool {}
```